### ISSUE TITLE
Graph panel legend 'current' should be changed to 'last'

### ISSUE THREAD

**Issue by nabelekt** [grafana/grafana#30046]
**What would you like to be added**:
The `current` label in the graph panel legend should be changed to `last`.

**Why is this needed**:
`current` assumes that we are working with live data and that the time range includes the current time. Both of these assumptions are often false.

**Comment by torkelo** [comment]
I agree

**Comment by simPod** [comment]
True. Doing in my backend already but have not ever questioned this in Grafana's legend. Makes sense 👍🏾

**Comment by simPod** [comment]
activity

**Comment by nabelekt** [comment]
I think this was already completed a long time ago?
